.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: ci
```

Comment:
`create-nuxt-app` autogenerated this CI config to run `eslint`. I'm undecided on whether we should keep it or it will be too much of a barrier for people who want to start contributing. What do you think @maxlapides ?

Reply:
I think we should keep it because ESLint does a great job providing code hints, which can be useful especially for newer JavaScript developers. I'd be happy to help anyone set this up in their local environment, but for most people they'll just need to install the ESLint plugin for their IDE, ex: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint